### ISSUE TITLE
"npm test" fails in two places

### ISSUE THREAD

**Issue by mlolson** [redfin/react-server#936]
I'm seeing tests fail in two places.  

1. The react-server-cli stdout tests in `commands.js` fail because React v15.6.1 throws a deprecation warning regarding proptypes. See: https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

2. react-server-integration tests in InternalServerErrorSpec fail because the tests expect a text response: '[object Object]', but the end point actually returns html that looks like this:

```
<!DOCTYPE html>
  <html lang="en">
    <head>
    <meta charset="utf-8">
    <title>Error</title>
  </head>
  <body>
    <pre>[object Object]</pre>
  </body>
</html>
```

Repro:

1. Clone react-server, 

2. Run  `npm install`

3. `npm test`

**Comment by gigabo** [comment]
@mlolson Was this fixed by #937?

**Comment by mlolson** [comment]
Yes I believe so.
